ui/app/components/mfa/mfa-login-enforcement-form.js

```diff
@@ -59,6 +60,8 @@ export default class MfaLoginEnforcementForm extends Component {
     this.resetTargetState();
     // only auth method types that have mounts can be selected as targets -- fetch from sys/auth and map by type
     this.fetchAuthMethods();
+    // this.args.model.mfa_methods returns a PromiseManyArray. We need to resolve this before using it in the template.
```

Comment:
nit: I feel like the comment below is sufficient and we don't need this here as well (it feels self-explanatory to me!) No need to remove, but just my thoughts 💭